app/code/UpdateDocsCronTask.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+class UpdateDocsCronTask implements CronTask {
```

Comment:
Rename to DocsCronTask since, in the future, it may do other things in addition to updating markdown files ? :)

@mandrew changed the name for docs so perhaps for consistency keep the name the same ?